lib/Service/BotService.php

```diff
@@ -51,6 +54,36 @@ public function __construct(
 	) {
 	}
 
+	public function afterBotEnabled (BotEnabledEvent $event): void {
+		$this->sendAsyncRequest($event->getBotServer(), [
+			'type' => 'Join',
+			'actor' => [
+				'type' => 'Bot',
+				'name' => $event->getBotServer()->getName(),
+			],
+			'object' => [
+				'type' => 'Collection',
+				'id' => $event->getRoom()->getToken(),
+				'name' => $event->getRoom()->getName(),
+			],
+		]);
+	}
+
+	public function afterBotDisabled (BotDisabledEvent $event): void {
+		$this->sendAsyncRequest($event->getBotServer(), [
+			'type' => 'Leave',
+			'actor' => [
+				'type' => 'Bot',
```

Comment:
Same as above:
According to https://www.w3.org/TR/activitystreams-vocabulary/#actor-types it should be Application I guess.
And we can add the id as well:
```suggestion
				'type' => 'Application',
				'id' => Attendee::ACTOR_BOTS . '/' . Attendee::ACTOR_BOT_PREFIX . $event->getBotServer()->getUrlHash()
```